rest-api/dao/physical_measurements_dao.py

```diff
@@ -46,15 +47,33 @@ def get_with_session(self, session, obj_id, **kwargs):
       ParticipantDao().validate_participant_reference(session, result)
     return result
 
-  def get_with_children(self, physical_measurements_id):
+  def get_with_children(self, physical_measurements_id, for_update=False):
     with self.session() as session:
       query = session.query(PhysicalMeasurements) \
```

Comment:
You could just have:

return self.get_with_children_with_session(session, physical_measurements_id, for_update)

Then you don't duplicate the logic below.

Reply:
done